portfolio/src/main/webapp/recording.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Recording music</title>
+    <link rel="stylesheet" href="styleRecording.css">
+    <script src="recording.js"></script>
+  </head>
+  <body>
+    <div id = "intro">
+      <h1>Recording music</h1>
+      <p> One of my dreams is to learn to play as many instruments as possible,
+          and become an awesome one man band.
+          For now, I can play:
+      <button onclick="toggleKaval">Kaval with Guitar</button>
```

Comment:
`toggleKaval` doesn't seem to be defined. What should happen when this button is clicked?